rtc_auth_enclave/Makefile

```diff
@@ -31,7 +31,8 @@ CUSTOM_BUILD_PATH := build
 CUSTOM_LIBRARY_PATH := $(CUSTOM_BUILD_PATH)/lib
 CUSTOM_BIN_PATH := $(CUSTOM_BUILD_PATH)/bin
 
-Crate_Files := $(wildcard src/*.rs)
+# XXX: Is there a better way to express this source dependency on rtc_tenclave?
```

Comment:
This seems to already be changed in main. Am I missing something?

Reply:
@PiDelport I think you might have a better idea of what @longtomjr is asking

Reply:
(This was just being pulled in by an old reference, I think: not in the current branch anymore.)